namespaces/api-namespace.md

```diff
@@ -42,6 +42,11 @@ For example a `category` element may be classified both as `resourceGroup` and
             - resourceGroup - Category is a set of resource.
             - dataStructures - Category is a set of data structures.
             - scenario - Reserved. Category is set of steps.
+- `attributes`
+    - `meta` (object) - Arbitrary metadata
```

Comment:
A couple of things:

1. I'm not sure I see that big of a difference myself between `meta` and `metadata`
1. Since we've been using camel case, shouldn't it be `metaData`?
1. In Minim, we will be providing convenience properties for meta attributes, meaning you'll use `element.id` and `element.title` for accessing `element.meta.id` and so on. This may shield the user enough from this issue.